packages/xfsprogs/xfsprogs.spec

```diff
@@ -63,6 +64,8 @@ rm -f %{buildroot}/%{_cross_libdir}/*.{la,a}
 %{_cross_sbindir}/xfs_repair
 %{_cross_sbindir}/xfs_rtcp
 %{_cross_sbindir}/xfs_spaceman
+%{_cross_sbindir}/xfs_property
```

Comment:
This is a shell script, and should be excluded. If the functionality is useful, we'll need to integrate it into `xfscli`, but I wouldn't bother without a clear use case.

It ends up calling into either `xfs_io` or `xfs_db`, so it roughly follows the pattern of what `xfscli` supports.